### PR TITLE
PFX-234 - Resolve warnings in Docusaurus build process

### PR DESCRIPTION
**Pull request by mmason2-godaddy**
## Summary
[PFX-234](https://jira.godaddy.com/browse/PFX-234) -  Resolve warnings in Docusaurus build process

## Changelog
- add LICENSE.md to `docsSetupDefault` in the docs plugin

### CLI verified via - [6.23.1-canary.1](https://www.npmjs.com/package/@gasket/plugin-docusaurus/v/6.23.1-canary.1)
Screen Shot 2022-06-02 at 3 11 07 PM

### Broken links resolved
Screen Shot 2022-06-02 at 3 11 50 PM

### Tests added
Screen Shot 2022-06-02 at 3 09 22 PM

### CODE REVIEW

File: packages/gasket-plugin-docs/lib/get-commands.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ module.exports = function getCommands(gasket, { GasketCommand, flags }) {
       }
       docsConfigSet.guides.unshift(...guides);
       await generateIndex(docsConfigSet);
+      await generateLicense(docsConfigSet);
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-docs/lib/get-commands.js:34]: Hmm, I think you're do more work than is necessary. Couldn't you just add `LICENSE.md` to the docSetup files that it just gets captured along with the readme, etc.?

```diff
module.exports = function docsSetup() {
  return {
    link: 'README.md',
    files: [
      'README.md',
+     'LICENSE.md',
      'docs/**/*'
    ],
    transforms: [
      txGasketPackageLinks,
      txGasketUrlLinks,
      txAbsoluteLinks
      txAbsoluteLinks
    ]
```

File: packages/gasket-plugin-docs/lib/docs-setup.js [hunk 2]
```diff
@@ -19,7 +20,8 @@ module.exports = function docsSetup() {
     transforms: [
       txGasketPackageLinks,
       txGasketUrlLinks,
-      txAbsoluteLinks
+      txAbsoluteLinks,
+      txLicenseLinks
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-docs/lib/docs-setup.js:24]: If you just include the LICENSE.md in the files is this extra transform necessary?

File: packages/gasket-plugin-docs/lib/get-commands.js (hunk 1 above)
Comment (mmason2-godaddy) [reply packages/gasket-plugin-docs/lib/get-commands.js:34]: Yeah I can do that. I didn't know that's how that behaved.

File: packages/gasket-plugin-docs/lib/docs-setup.js (hunk 2 above)
Comment (mmason2-godaddy) [reply packages/gasket-plugin-docs/lib/docs-setup.js:24]: I believe so but I'll doublecheck. All the links to the License file are broken either with the file present.

**Review by agerard-godaddy — APPROVED** [review]

**Review by agerard-godaddy — APPROVED** [review]

**Review by jpage-godaddy — APPROVED** [review]

**Pull request merged**